packages/web/server/api/edge-routers/pools-router.ts

```diff
@@ -35,9 +35,9 @@ export const poolsRouter = createTRPCRouter({
     .input(
       GetInfinitePoolsSchema.and(
         z.object({
-          sort: createSortSchema(marketIncentivePoolsSortKeys)
-            .optional()
-            .default({ keyPath: "totalFiatValueLocked" }),
+          sort: createSortSchema(marketIncentivePoolsSortKeys).default({
+            keyPath: "totalFiatValueLocked",
+          }),
```

Comment:
Following my earlier comment, I think this change won't cause any issues; instead, it will set the default value as intended. Previously, the value remained undefined, which negated the purpose of having a default.